cycledash/static/css/style.css

```diff
@@ -10,3 +10,29 @@ section {
   margin: auto;
 }
 img.cycle { float: right; }
+
+.comments {
+  list-style: none;
+}
+.comments .run-id {
+  font-weight: bold;
+}
+.comments .run-id a {
+  color: black;
+}
+.comments .location {
+  padding-left: 7px;
+}
+.comments .summary {
+  padding-left: 20px;
+}
+.comments .summary:before {
```

Comment:
The padding side {left, right} actually differs between the two, so I figured I'd just duplicate the font-size in both and save a line/extra rule.